lib/executor/helpers.go

```diff
@@ -101,7 +100,6 @@ func getIterationRunner(
 				} else {
 					logger.Error(err.Error())
 				}
-				//TODO: investigate context cancelled errors
```

Comment:
I don't think this needed to be removed ... IMO this select will be wrong in some cases where the iteration actually finished but the context finished after it but before we got to here ... And given that if the iteration was interrupted we have an error, I would prefer if we actually look at the error, and if the error is "interrupted" to then increase the Interrupted counter.

Reply:
I'm not entirely sure if that scenario can happen, but I removed this at @na--'s suggestion in https://github.com/loadimpact/k6/pull/1368#discussion_r400160573, and now it's back in 5815716e.

It's just a comment anyway, but let me know.